charts/sealed-secrets/templates/03-update-crd-job.yaml

```
@@ -0,0 +1,58 @@
+{{- /*
+MIT License
+
+(C) Copyright 2025 Hewlett Packard Enterprise Development LP
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+*/}}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "sealed-secrets-pre-upgrade-crd-hook"
+  annotations:
+    helm.sh/hook: pre-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+spec:
+  ttlSecondsAfterFinished: 86400
+  template:
+    metadata:
+      name: "sealed-secrets-pre-upgrade-crd-hook"
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: "sealed-secrets"
+      volumes:
+        - name: "sealed-secrets-crd-configmap"
+          configMap:
+            name: "sealed-secrets-crd-configmap"
+            defaultMode: 0755
+      restartPolicy: Never
+      containers:
+        - name: sealed-secrets-pre-upgrade-crd-hook
+          image: "{{ .Values.kubectl.image.repository }}:{{ .Values.kubectl.image.tag }}"
+          imagePullPolicy: "{{ .Values.kubectl.image.pullPolicy }}"
+          volumeMounts:
+           - name: "sealed-secrets-crd-configmap"
+             mountPath: /tmp
+          command:
+            - '/bin/sh'
+          args:
+            - "-c"
+            - kubectl apply -f /tmp/bitnami.com_sealedsecrets.yaml
```

Don't mount over `/tmp`, that's just not good practice.  Use something like `/crds` instead.
```suggestion
             mountPath: /crds
          command:
            - '/bin/sh'
          args:
            - "-c"
            - kubectl apply -f /crds/bitnami.com_sealedsecrets.yaml
```